DPPP/BDABuffer.h

```diff
@@ -196,7 +193,7 @@ namespace DP3 {
       }
 
       static constexpr bool TimeIsEqual(double x, double y) {
-        return abs(x - y) < kTimeEpsilon;
+        return fabs(x - y) < kTimeEpsilon;
```

Comment:
(std::abs is fine, see https://stackoverflow.com/questions/3118165/when-do-i-use-fabs-and-when-is-it-sufficient-to-use-stdabs , in either case add std::)

Reply:
Since abs and fabs are not constexpr, I already replaced them by `((x > y) ? (x - y) : (y - x))`.